### PR TITLE
Update Jetpack onboarding flow: Update instructions for logged out (new) users

### PR DESCRIPTION
**Pull request by manzoorwanijk**
#### Proposed Changes

* This PR updates the instructions on the Thank You page for logged out (new) users to make them the same as the manual installation option (for logged in users).

#### Testing Instructions

- Do any one of these
    - Open Calypso Live link below and wait for the redirect to complete
    - or boot up this PR 
        - Run `git fetch && git checkout update/jp-onboarding/instructions-for-logged-out-users`
        - Run `yarn start`
* Open a new incognito/private browser window and go to https://cloud.jetpack.com/pricing
* Purchase a product like Backup or Boost
* Complete the checkout with a new email address that does not have a WPCOM associated with it. You can add an alias to your email address with `+something` e.g. `username+jptest@something.com`
* If you want to complete the checkout for free, please check this Slack conversation: p1666675278864859-slack-C04457PNH1D. It will work for Backup and Boost
* On the Thank You page, replace `https://wordpress.com` with `http://calypso.localhost:3000` or with the Calypso Live link above
* Add `&flags=jetpack/standalone-plugin-onboarding-update-v1` query parameter to the URL to activate the feature flag.
* Confirm that you see the new instructions

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

Completes 1203097885147382-as-1203159154260043/f

Screenshot 2022-10-26 at 4 14 26 PM

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1203159154260043

### CODE REVIEW

File: client/my-sites/checkout/checkout-thank-you/licensing-thank-you-manual-activation-instructions.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ const LicensingActivationInstructions: FC< JetpackLicenseKeyProps > = ( {
 				}
 				footerImage={ licensingActivationPluginInstall }
 				showContactUs
-				showProgressIndicator
+				showProgressIndicator={ false }
```
Comment (jkguidaven) [inline client/my-sites/checkout/checkout-thank-you/licensing-thank-you-manual-activation-instructions.tsx]: @manzoorwanijk for the other products such as the bundles. We actually still display the old flow which the license activation instruction is in a separate page.

should we maybe check here if we are trying to display a standalone installation?

```
showProgressIndicator={ ! jetpackStandaloneProduct }
```

Comment (manzoorwanijk) [reply client/my-sites/checkout/checkout-thank-you/licensing-thank-you-manual-activation-instructions.tsx]: That is a nice suggestion. Thanks

File: client/my-sites/checkout/index.js [hunk 2]
```diff
@@ -56,7 +55,7 @@ export default function () {
 	page(
 		'/checkout/jetpack/thank-you/licensing-manual-activate/:product',
 		noSite,
-		licensingThankYouManualActivation,
```
Comment (jkguidaven) [inline client/my-sites/checkout/index.js:59]: Just one question here, if this gets deployed. the checkout flow in the production will be affected immediately.

Do we need to use `jetpack/standalone-plugin-onboarding-update-v1` feature flag here?

**Review by jkguidaven — APPROVED** [review]
Looks good to me other than just the one question. 👍

File: client/my-sites/checkout/index.js (hunk 2 above)
Comment (manzoorwanijk) [reply client/my-sites/checkout/index.js:59]: `LicensingActivationInstructions` already uses the feature flag, so I think it's good to go.

**Pull request merged**